core/src/hash/accumulator.rs

```diff
@@ -113,3 +131,16 @@ impl<'a> Accumulator for Wrap<'a> {
         &self.buffer[..self.len]
     }
 }
+
+#[cfg(not(feature = "std"))]
+impl<'a> scale::Output for Wrap<'a> {
+    fn write(&mut self, bytes: &[u8]) {
+        <Self as Accumulator>::write(self, bytes)
+    }
+
+    fn push_byte(&mut self, byte: u8) {
+        assert!(self.len() < self.capacity());
```

Comment:
As above. I think this is just redundant noise (IMHO).

Reply:
true